### ISSUE TITLE
Consider hooking form validation exceptions into global.php

### ISSUE THREAD

**Issue by ninthspace** [ninthspace/larabase#14]
Then they don't need to be tried and caught everywhere

**Comment by ninthspace** [comment]
For example:

```php
App::error(function (Laracasts\Validation\FormValidationException $exception, $code) {
    $count = count($exception->getErrors());
    $tense = "was";
    if ($count != 1) {
        $tense = "were";
    }
    Flash::bad("There " . $tense . " " . str_pluralize($count, 'error', TRUE) . " in your form, please correct it");

    return Redirect::back()->withInput()->withErrors($exception->getErrors());
});

**Comment by ninthspace** [comment]
Closed with #13.
